docs/example_workflow_explained.rst

```diff
@@ -0,0 +1,316 @@
+    
+    # Here we clean up a few naming conventions to get everything on the same page.
+    if 'hru_id2' in basin_topo:
+        basin_topo['hru'] = basin_topo['hru_id2']
+    if 'seg_id' in basin_topo:
+        basin_topo['seg'] = basin_topo['seg_id']
+
+
+
+
+Convert ``mizuroute`` formatting to ``bmorph`` formatting
+---------------------------------------------------------
+
+``mizuroute_utils`` is our utility script that will handle converting
+Mizuroute outputs to what we need for ``bmorph``. For more information
+on what ``mizuroute_utils`` does specifically and how to change its 
+parameters, check out ``data.rst``.
+
+Here we pull out coordinate data from the ovearching watershed
+for the specific basin we want to analyze.
+    
+.. code:: ipython3 
+    
+    basin_ref = watershed_ref.sel(site=[r for r in ref_sites])
+    
+    for site, seg in site_to_seg.items():
+        if site in basin_ref['site']:
+            basin_ref['seg'].loc[{'site': site}] = seg
+    
+Now we pass it off to ``mizuroute_to_blendmorph``, the primary utility 
+function for automating ``bmorph`` pre-procesing.
+    
+.. code:: ipython3 
+
+    basin_met_seg = mizutil.mizuroute_to_blendmorph(
+        basin_topo, watershed_raw.copy(), basin_ref, watershed_met, 
+        fill_method='r2').ffill(dim='seg')
+
+Apply ``bmorph`` bias correction
+--------------------------------
+
+We are almost to actually bias correcting! First we need to specify some parameters 
+for correction. Returning to these parameters can help fine tune your bias 
+corrections to the basin you are analyzing.
+
+In this notebook, all four variations of ``bmorph`` are demonstrated: 
+IBC_U, IBC_C, SCBC_U, and SCBC_C, as described in ``bias_correction.rst``.
+
+The ``train_window`` is what we will use to train the bias correction
+model. This is the time range that is representative of the
+basin's expected behavior that ``bmorph`` should strive to mirror.
+
+The ``bmorph_window`` is when ``bmorph`` should be applied to the series for
+bias correction.
+
+Lastly the ``reference_window`` is when the reference flows should be used to 
+smooth the bias corrected flows. This is recommended to be set as equivalent to the
+``train_window``.
+    
+.. code:: ipython3 
+
+    train_window = pd.date_range('1981-01-01', '1990-12-30')[[0, -1]]
+    bmorph_window = pd.date_range('1991-01-01', '2005-12-30')[[0, -1]]
+    reference_window = train_window
+    
+``interval`` is the length of``bmorph``'s application intervals, 
+typically a factor of years to preserver hydrologic 
+relationships. Note that for ``pandas.DateOffset``, 'year' and 'years' 
+are different and an 's' should always be included here for ``bmorph``
+to run properly, even for a single year.
+
+``overlap`` describes how many days the bias correction cumulative distribtuion function
+windows should overlap in total with each other. ``overlap`` is evenly distributed before
+and after this window.
```

Comment:
We might want to note this is to reduce the amount of "jaggedness" (for lack of a better term) on the application intervals.

Reply:
Perhaps "discontinuities between application windows"?